proxy/ProxySession.h

```diff
@@ -151,9 +148,7 @@ class ProxySession : public VConnection
 
   IpAllow::ACL acl; ///< IpAllow based method ACL.
 
-  IpAddr outbound_ip4;        ///< Local address for outbound connection.
-  IpAddr outbound_ip6;        ///< Local address for outbound connection.
-  in_port_t outbound_port{0}; ///< Local port for outbound connection.
+  HttpSessionAccept::Options const *accept_options; ///< connection info // L7R TODO: set in constructor
```

Comment:
I'm curious why this change (not this line but whole change) is not a part of #5805.

The new interface sounds good to me, but do we need to do this separately from #5805 at this time? I'm worried about conflicts on backporting other changes.